server/routerlicious/packages/lambdas/src/deli/lambda.ts

```diff
@@ -198,7 +199,8 @@ export class DeliLambda extends TypedEventEmitter<IDeliLambdaEvents> implements
         private readonly reverseProducer: IProducer,
         private readonly serviceConfiguration: IServiceConfiguration,
         private sessionMetric: Lumber<LumberEventName.SessionResult> | undefined,
-        private sessionStartMetric: Lumber<LumberEventName.StartSessionResult> | undefined) {
+        private sessionStartMetric: Lumber<LumberEventName.StartSessionResult> | undefined,
+        globalDbMongoManager?: MongoManager) {
```

Comment:
Do we need to pass `MongoManager` here? The lambdas used to be very reliant on `DocumentCollection` / Mongo related methods and we explicitly that coupling. We made it so database related changes went through interfaces & event emitters. 
For example, today we pass an `IDeliCheckpointManager` and that can do whatever you want behinds the scenes. 

In this scenario, it sounds like you will do something when the session is closed. Will that involve executing some code when the deli `close` method is called? Instead of passing a `MongoManager`, can you use an event. It could fire a "close" event and you can listen to that inside the lambda factory or somewhere else outside the deli class itself.